.github/workflows/create-new-page.yml

```diff
@@ -0,0 +1,37 @@
+name: Create a new web page 
+
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: "Name of an existing branch for work on the new page"
+        required: true
+        type: string
+      page:
+        description: "Filename to use for the new page (.md will be added)"
```

Comment:
Good point, and since I now know this string will be the permalink (URL) as well as filename, I'm going to reword the prompt and limit to letters, digits, dashes. Thanks!